packages/salesforcedx-aura-language-server/scripts/transform-aura-system.js

```diff
@@ -0,0 +1,31 @@
+// NOTE: This script is not used in the build.js, but needs to be run manually to update the aura-system.json file
```

Comment:
do we use this?

ex: it's possible that the "original" `aura-system.json` comes from some other place (ex: core, or some other closed repo).  If so, we probably want to use that as the checked in file and have `transformed-aura-system.json` as gitignored, and then have the transform run automatically somewhere in the compile step